packages/react-magma-dom/src/components/Datagrid/Datagrid.stories.tsx

```diff
@@ -337,18 +343,23 @@ const defaultArgs = {
 };
 
 export const Default = Template.bind({});
-Default.args = defaultArgs;
+Default.args = { ...defaultArgs, tableTitle: 'Default' };
```

Comment:
`tableTitle: 'Default'` should be moved inside `defaultArgs` where all the default arguments are